modules/sdk/gradle-plugins-workspace/src/gradleTest/targetPlatformDefault/build.gradle

```diff
@@ -1,10 +1,18 @@
+allprojects {
```

Comment:
Please follow [these guidelines](https://github.com/liferay/liferay-portal/blob/master/modules/README.markdown#gradle-files), `task ...` should come before `allprojects`.